### ISSUE TITLE
UI regression: login spinner is rendered over ellipse rather than circle

### ISSUE THREAD

**Issue by patkovskyi** [okta/okta-signin-widget#1381]
### I'm submitting a
  - [x] bug report
  - [ ] feature request

### Background info
We migrated from version 3.9.2 to 4.3.2 and noticed a UI regression.

### Expected behavior
No UI imperfections.

### What went wrong?
A round spinner is now rendered over an elliptical background (which is definitely not by design).

### Steps to reproduce
We simply migrated our application from 3.9.2 to 4.3.2.
Here's our `SignInWidget.tsx`:

```
import React from 'react';
import OktaSignIn from '@okta/okta-signin-widget';
import '@okta/okta-signin-widget/dist/css/okta-sign-in.min.css';

import getAppConfig from '../../config/getAppConfig';

export default class extends React.Component {
  widget: typeof OktaSignIn | undefined;

  componentDidMount() {
    try {
      const { baseUrl, issuer, clientId, redirectUri } = getAppConfig().okta;
      this.widget = new OktaSignIn({
        baseUrl,
        el: '#okta-widget-root',
        logo: '/img/logo/logo.png',
        brandName: 'SomeBrand',
        redirectUri,
        authParams: {
          display: 'page',
          responseType: 'code',
          grantType: 'authorization_code',
          issuer,
        },
      });

      const renderOptions = {
        el: '#okta-widget-root',
        clientId,
        redirectUri,
        authParams: {
          issuer,
          display: 'page',
          responseMode: 'query',
          responseType: ['id_token', 'token'],
          scopes: getAppConfig().okta.scopes,
        },
      };

      this.widget.renderEl(renderOptions);
    } catch (e) {
      // eslint-disable-next-line no-console
      console.log('Okta Widget Init Error', e);
    }
  }

  componentWillUnmount() {
    try {
      if (this.widget) this.widget.remove();
    } catch (e) {
      // eslint-disable-next-line no-console
      console.log(e);
    }
  }

  render() {
    return <div id="okta-widget-root" />;
  }
}
```

### Your environment
  - Okta Sign-In Widget Version: 4.3.2
  - Browser: Chrome 86.0.4229.0
  - OS: MacOS Catalina 10.15.6 (19G2021)
  - Language: English

### Additional info
I set some DOM breakpoints and figured out that it happens because the `beacon-blank` class is set on one of the divs created for this animation. https://github.com/okta/okta-signin-widget/blob/05b8b2b978ac9f0df82fabf1e327677bf83d4c6d/assets/sass/modules/_beacon.scss#L16-L19
When I unset these width/height properties the UI is fixed. See screenshots.

![Screenshot 2020-08-19 at 17 18 00](https://user-images.githubusercontent.com/1560240/90647029-8bd50800-e240-11ea-8893-07e24051106f.png)
![Screenshot 2020-08-19 at 17 19 06](https://user-images.githubusercontent.com/1560240/90647047-9099bc00-e240-11ea-9057-080fa5d67aa0.png)
![Screenshot 2020-08-19 at 17 19 19](https://user-images.githubusercontent.com/1560240/90647052-92637f80-e240-11ea-9883-14fab6062441.png)

**Comment by aarongranick-okta** [comment]
@patkovskyi Thanks for the report. We have noticed a few minor CSS regressions in the current version. We are working on getting these fixed quickly. In the meantime you should be able to work around the issue by adding some CSS to your page which overrides these bad styles. 

Internal ref: OKTA-323553

**Comment by patkovskyi** [comment]
We upgraded to 4.5.1 and this does not happen anymore.

**Comment by haishengwu-okta** [comment]
https://github.com/okta/okta-signin-widget/pull/1411
Fixed since 4.4.2
